modules/mobile/spec/support/fixtures/VAOS_v2_appointments.json

```diff
@@ -41,6 +41,9 @@
     "location":{
       "id":"442",
       "name":"Cheyenne VA Medical Center",
+      "timezone" : {
+        "timeZoneId" : "America/Denver"
```

Comment:
used mock location adapter test data in which only had fields that were being used. so had to add in timezone.